wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Field.kt

```diff
@@ -209,10 +215,28 @@ class Field private constructor(
     OPTIONAL,
     REQUIRED,
     REPEATED,
+
     /** Indicates the field is a member of a `oneof` block.  */
     ONE_OF
   }
 
+  enum class EncodeMode {
+    /** Optional from proto2. */
+    NULL_IF_ABSENT,
+
+    /** Required from proto2. */
+    THROW_IF_ABSENT,
+
+    /** Non-repeated fields in proto3. */
```

Comment:
Can you say `0`, `false`, or `null` here?

Reply:
fixed, and added `""` as well.